data/nation/congofreestate.ruleset

```diff
@@ -0,0 +1,89 @@
+[nation_congofreestate]
+
+name=_("Congo Free State")
+plural=_("?plural:Congo Free States")
+groups="Modern", "African"
+
+legend=_("The Congo Free State was a large state in Central Africa from 1885 to 1908\
+ which was in personal union with the Kingdom of Belgium under Leopold II.\
+ Leopold's reign in the Congo eventually earned infamy due to the increasing\
+ mistreatment of the indigenous peoples.")
+
+leaders = {
+ "name",                        "sex"
+ "Leopold II",                  "Male"	
+ "Marie Henriette",             "Female"
+ "Francis Walter de Winton",    "Male"
+ "Camille Janssen",             "Male"
+ "Théophile Wahis",             "Male"
+ "Joseph Conrad",               "Male"
+ "Henry Morton Stanley",        "Male"
+ "Tippu-Tip",                   "Male"
+}
+
+ruler_titles = {
+ "government",      "male_title",                     "female_title"
+ "Despotism",       _("Governeror-General %s"),      _("?female:Governeror-General %s")
+}
+
+flag="congofreestate"
+flag_alt = "german"
+style = "Tropical"
+
+init_techs=""
+init_buildings=""
+init_units=""
+conflicts_with = "kongo", "congolese"
+civilwar_nations = "katangan", "rwandan", "burundian", "kongo", "congolese", "congolesebrazzaville"
```

Comment:
I don't think Congo-Brazzaville should be a civil war nation for the Congo Free State, as it was a French colony unrelated in everything but the name.